airframe-http-recorder/README.md

```diff
@@ -44,30 +64,35 @@ withResource(HttpRecorder.createReplayOnlyServer(recorderConfig)) { server =>
   val addr = server.localAddress // "localhost:(port number)"
   // Requests to the local server will return the recorded responses
 }
+
 ```
 
-### Programmable
+### Programmable HttpRecorderServer
 
 ```scala
 import wvlet.airframe.http.recorder._
 import wvlet.airframe.control.Control._
 import com.twitter.finagle.http.{Request,Response}
 
-val response = withResource(HttpRecorder.createProgrammableServer { recorder =>
-  // Program server responses instead of recodring
+val recorderConfig = HttpRecorderConfig(sessionName="my-recording")
+val response = withResource(HttpRecorder.createProgrammableServer(recorderConfig)) { server =>
+  server.start
```

Comment:
By the way, why do we need to start the server in our code?

Reply:
I also thought this is a bit annoying. We can start the server within createXXX methods for convenience.